.gitlab/ci/global.yml

```diff
@@ -71,8 +71,175 @@
     - demisto-sdk --version
     - section_end "Build Parameters"
 
+.default-after-script:
+  after_script:
+    - source .gitlab/helper_functions.sh
+    - section_start "Notify Slack"
+    - |
+      if [ -n "${SLACK_ONLY_IF}" ]; then
+        ./Tests/scripts/slack_notifier.sh "$SLACK_TEST_TYPE" "$SLACK_ENV_RESULTS" "$SLACK_CHANNEL"
+      else
+        echo "Skipping Slack"
+      fi
+    - section_end "Notify Slack"
+    - section_start "Destroy instances"
+    - |
+      if [ -n "${DESTROY_INSTANCES}" ]; then
+        python3 ./Tests/scripts/destroy_instances.py $ARTIFACTS_FOLDER $ARTIFACTS_FOLDER/env_results.json "$INSTANCE_ROLE" "$TIME_TO_LIVE"
+      fi
+    - section_end "Destroy instances"
 
 .default-job-settings:
   extends:
     - .default-cache
     - .default-before_script
+    - .default-after-script
+
+.run-unittests-and-lint:
+  needs: []
+  stage: unittests-and-validations
+  artifacts:
+    expire_in: 48 hrs
```

Comment:
Do we want these artifacts to expire after 48 hours for all workflows where this job may run? Is that the behavior we have in CircleCI? Same question for `.run-validations`.

Reply:
In previous meetings it was decided on 30 days, changed it now